src/components/projects/ProjectCardLeft.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import Image from "next/image";
+
+const ProjectCardLeft = ({ projectName, description }) => {
+  return (
+    <div className="flex flex-col-reverse md:flex-row items-center md:items-start">
+      {/* Text below the image on small screens */}
+      <div className="mt-4 md:mt-0 md:mr-4 w-full max-w-xs">
+        <p className="text-xl md:text-2xl text-black pt-14 pl-2">
+          {description}
+        </p>
+      </div>
+
+      {/* Image container */}
+      <div className="relative w-full md:w-auto">
+        <Image
+          src="/images/projectCard.webp"
+          alt=""
+          width={833}
+          height={436.31}
```

Comment:
import image statically on top, `import projectCard from @/public/projectCard.webp` then use that as the src, `src={projectCard}` and remove static heights and width